src/components/guide.jsx

```diff
@@ -86,17 +86,20 @@ const Guide = forwardRef((props, ref) => {
   const readyToProceed = (newStepIndex) => {
     if (newStepIndex < steps.length) {
       const stepName = steps[newStepIndex].step;
-      if (stepName === 'addAnotherColumn') {
+      if (stepName === 'selections') {
+        const selections = document.querySelector('.selections-inner li');
+        if (!selections) return false;
+      } else if (stepName === 'selectEntity' || stepName === 'selectAnotherEntity') {
+        const overlay = document.getElementsByClassName('cube-column-chooser');
+        if (!overlay.length > 0) {
```

Comment:
This looks odd, `!overlay.length` will return a bool, and then we compare that to `> 0` (`false` == `0`, `true` == `1`), did you mean `if (overlay.length) return false;`?

Reply:
Oops, a refactor that didn't go all the way. I fix :)